src/pybind/mgr/mgr_module.py

```diff
@@ -40,6 +42,25 @@
     "creating",
     "unknown"]
 
+def is_ipv6_enabled():
+    """
+    Check whether IPv6 is enabled.
+    :return: Return True if IPv6 is enabled, otherwise False.
+    :rtype: bool
+    """
+    if not os.path.exists('/proc/net/if_inet6'):
+        return False
+    with open('/proc/net/if_inet6') as f:
+        lines = f.readlines()
+    # Filter unwanted interfaces.
+    lines = [l for l in lines if not re.match(r'^\s+lo$', l)]
+    return len(lines) > 0
+
+if is_ipv6_enabled():
+    DEFAULT_ADDR = '::'
```

Comment:
I still don't like this constant, as it is constant. Can you call `is_ipv6_enabled()` directly within Dashboard's `serve()`?

Reply:
DEFAULT_ADDR were used in dashboard::CherryPyConfig, dashboard:Module, prometheus:Module, prometheus:StandbyModule , if is_ipv6_enabled() called in Dashboard's serve(), others module have to call it too, define it as a constant here, more easy for modules to use, what do you think.

Reply:
Well, one big downside is: we're now calling is_ipv6_enabled for every import of every module, even if the module doesn't make any use of it. 

ideas?

Reply:
```
if is_ipv6_enabled():
    DEFAULT_ADDR = '::'
```
move DEFAULT_ADDR to dashboard and prometheus as original？

Reply:
we might want to move this function to `src/pybind/mgr/mgr_util.py`, and if we can use a function instead a global variable for retrieving `DEFAULT_ADDR`, we can even memorize it, like
```python
def get_default_addr():
    def is_ipv6_enabled():
        #
        # ...
    try:
        return get_default_addr.result
    except AttributeError:
        result = '::' if is_ipv6_enabled() else '0.0.0.0'
        foo.result = result
        return get_default_addr.result
```